tasks/main.yml

```diff
@@ -3,6 +3,9 @@
   apt: pkg=ntp={{ ntp_version }} state=present
 
 - name: Configure NTP
-  template: src=ntp.conf.j2 dest=/etc/ntp.conf
+  template: src={{ item }} dest=/etc/ntp.conf
+  with_first_found:
+    - "ntp.{{ ansible_distribution_release }}.conf.j2"
+    - "ntp.conf.default.j2"
```

Comment:
I did a little bit of research and reading of files, and I believe the default file will work on both distributions. The difference between the two configurations is that the file for xenial uses the new `pool` configuration directive, and including some the management methods outlined [here](https://www.ntppool.org/join/configuration.html). The trusty/default config uses the older `server` directive instead, but according to [this article] (https://www.eecis.udel.edu/~mills/ntp/html/discover.html#pool), newer versions of NTP are backwards compatible so `server` will still work. I don't imagine that this would change across distributions.